### ISSUE TITLE
chore: allow release please to sign commits

### ISSUE THREAD

**Issue by weboko** [waku-org/js-waku#1247]
This is a **support request**

## Problem

Currently we had to remove hard rule for all commits to be signed to unblock the release https://github.com/waku-org/js-waku/pull/1237

## Proposed Solutions

We need to add GPG token to allow `release-please` to sign it's commits 
https://github.com/peter-evans/create-pull-request/blob/main/docs/concepts-guidelines.md#gpg-commit-signature-verification

**Comment by weboko** [comment]
Putting on hold for the time being since `release-please` underlying library does not support it and there is no workaround for now.

https://github.com/googleapis/code-suggester/issues/337

**Comment by IchordeDionysos** [comment]
@weboko doesn't code-suggester now support supplying a commit signer/signing commits?
https://github.com/googleapis/code-suggester/blob/main/src/github/create-commit.ts#L47-L55
